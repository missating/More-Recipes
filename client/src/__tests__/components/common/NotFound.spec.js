import React from 'react';
import toJSON from 'enzyme-to-json';
import NotFound from '../../../components/common/NotFound';

describe('Not found component', () => {
  it('Should render', () => {
    const wrapper = shallow(<NotFound />);
    expect(wrapper.find('div').length).toEqual(1);
    expect(toJSON(wrapper)).toMatchSnapshot();
  });
});

