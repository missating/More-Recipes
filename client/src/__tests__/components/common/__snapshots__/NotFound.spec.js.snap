// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Not found component Should render 1`] = `
<div
  className="container text-center"
  id="recipes"
>
  <h1
    style={
      Object {
        "color": "#000",
      }
    }
  >
     Are You Lost ? 
  </h1>
  <h5>
     This page does not exist (404)
  </h5>
</div>
`;
